charts/quantum-serverless/charts/gateway/templates/deployment.yaml

```diff
@@ -167,8 +165,8 @@ spec:
             - name: DATABASE_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.secrets.servicePsql.name }}
-                  key: {{ .Values.secrets.servicePsql.key.databasePassword }}
+                  name: postgresql
+                  key: password
```

Comment:
I don't know if this is due to the rebase @akihikokuroda but this change needs to remain as it is.

Reply:
For the local install, the secret for the password is created by the database install.  So we need a little tweak.  I added another value `databaseSecretName` to separate the name of the secret and the secret name for the password.